procrastinate/manager.py

```diff
@@ -123,13 +123,18 @@ def batch_defer_jobs(self, jobs: list[jobs_module.Job]) -> list[jobs_module.Job]
     def _defer_jobs_query_kwargs(self, jobs: list[jobs_module.Job]) -> dict[str, Any]:
         return {
             "query": sql.queries["defer_jobs"],
-            "task_names": [job.task_name for job in jobs],
-            "queues": [job.queue for job in jobs],
-            "priorities": [job.priority for job in jobs],
-            "locks": [job.lock for job in jobs],
-            "queueing_locks": [job.queueing_lock for job in jobs],
-            "args_list": [job.task_kwargs for job in jobs],
-            "scheduled_ats": [job.scheduled_at for job in jobs],
+            "jobs": [
+                (
+                    job.queue,
+                    job.task_name,
+                    job.priority,
+                    job.lock,
+                    job.queueing_lock,
+                    job.task_kwargs,
+                    job.scheduled_at,
+                )
```

Comment:
An namedtuple is a tuple, so you could even use a namedtuple and kwargs and it should still work :)

Reply:
Oh yes, thanks. Makes it much clearer.